internal/loader/configuration_setting_loader.go

```diff
@@ -197,15 +282,47 @@ func (csp *ConfigurationSettingLoader) CreateKeyValueSettings(ctx context.Contex
 	}
 
 end:
-	typedSetting, err := createTypedSettings(configMapSettings, isJsonContentTypeMap, csp.Spec.Target.ConfigMapData)
-	if err != nil {
-		return nil, err
+	return rawSettings, nil
+}
+
+func (csl *ConfigurationSettingLoader) getFeatureFlagSettings(ctx context.Context) (map[string]interface{}, error) {
```

Comment:
Shouldn't the return type be `map[string][]interface{}`?

Reply:
`map[string]interface{}` is used to maintain consistency with the type returned by `UnmarshalConfigMap`